1-getting-started.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `1-getting-started.ipynb` Cell 58 Line 2_
```python
print('this is a string')
print("this is also string")
```
typo: `this is also a string`